docs/Using-Fleet/configuration-files/README.md

```diff
@@ -199,6 +199,9 @@ spec:
       - secret: RzTlxPvugG4o4O5IKS/HqEDJUmI1hwBoffff
       - secret: JZ/C/Z7ucq22dt/zjx2kEuDBN0iLjqfz
 ```
+
+> Please note that `agent_options` defined in a team configuration are the only agent options applied to hosts enrolled in that team. The `agent_options` defined here should include any necessary agent options defined in your Organization settings that you want applied to hosts enrolled in the team as well. 
```

Comment:
I think this could be more concise...I get a little lost reading this. I think this language is not as clear as the language you used above (the stuff about the override). Could bring that down here!